analysis_options.yaml

```diff
@@ -1,2 +1,7 @@
 analyzer:
   strong-mode: true
+  errors:
```

Comment:
Please add a comment indicating that these are promoted to errors because they're errors within Google.

Reply:
Done